tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -100,8 +100,8 @@ suite('Git with ssh workflow', async () => {
         await testWorkspaceUtils.createWsFromDevFile(data);
         await driverHelper.navigateToUrl(workspacePrefixUrl + wsNameCheckPropagatingKeys);
         await ide.waitWorkspaceAndIde();
-        await driverHelper.wait(5000);
         await projectTree.openProjectTreeContainer();
+        await driverHelper.wait(TimeoutConstants.TS_DEBUGGER_CONNECTION_TIMEOUT);
```

Comment:
it's interesting to know why "TimeoutConstants.TS_DEBUGGER_CONNECTION_TIMEOUT" ?

Reply:
Yes. Maybe TS_SELENIUM_LOAD_PAGE_TIMEOUT constant fits better.